remoteappmanager/templates/page.html

```diff
@@ -95,36 +93,35 @@
 
     <aside class="main-sidebar">
       {% block sidebar %}
-      {% endblock %}
+      {% end %}
     </aside>
 
     <div class="content-wrapper">
       <!-- Main content -->
       {% block main %}
-      {% endblock %}
+      {% end %}
     </div>
   </div>
 
   <div id="error-dialog-container"></div>
 
   <script src="{{ static_url("dist/sharedDependencies.js") }}"></script>
 
+  <script type="text/javascript">
```

Comment:
In tornado we don't have the possibility to call `super ()` in blocks, moving this script out of the block `script_init` was the only solution that I found.